client/src/common/components/Dashboard/DashboardCurrent/index.tsx

```diff
@@ -0,0 +1,30 @@
+import React, { FunctionComponent } from 'react';
+
+// Store
+import { useAppSelector } from '../../../../app/hooks';
+
+// API
+import { useGetCurrLocationWeatherQuery } from '../../../../services/forecaApi';
+
+// Components
+import Widget from '../../Widget';
+
+// Styles
+import * as S from '../styles';
+
+const DashboardCurrent: FunctionComponent = () => {
+  const { query } = useAppSelector(state => state.location);
+
+  const { data, isLoading, isError } = useGetCurrLocationWeatherQuery(query, {
+    skip: !query
+  });
+
+  return (
+    <S.DashboardCurrent>
+      <h3>Current weather</h3>
+      <Widget color="primary" data={data && data.current} isLoading={isLoading} isError={isError} />
```

Comment:
is <Widget/> a shared component that will be used to show data from different queries?

Reply:
No, this one just shows current weather. DashboardForecast (or maybe I'll give it a different name) component will be used to show data from different queries, like hourly, three-hourly, daily and air quality.

Reply:
in this case why do you pass weather data to `Widget` through props? you could just use `useGetCurrLocationWeatherQuery` inside it

Reply:
Widget is a shared component, so depending on different data from props (currWeather, hourlyWeather, etc.) would render different results. For instance, in DashboardForecast I may have a list of widgets that each would show a specific hour's data from array of hourly data, which I would get from, say, useGetLocationHourlyWeatherQuery.